data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/parser/LogstashVisitor.java

```diff
@@ -0,0 +1,170 @@
+package org.opensearch.dataprepper.logstash.parser;
+
+import org.opensearch.dataprepper.logstash.exception.LogstashParsingException;
+import org.opensearch.dataprepper.logstash.LogstashBaseVisitor;
+import org.opensearch.dataprepper.logstash.LogstashParser;
+import org.opensearch.dataprepper.logstash.model.LogstashConfiguration;
+import org.opensearch.dataprepper.logstash.model.LogstashPlugin;
+import org.opensearch.dataprepper.logstash.model.LogstashAttribute;
+import org.opensearch.dataprepper.logstash.model.LogstashAttributeValue;
+import org.opensearch.dataprepper.logstash.model.LogstashPluginType;
+import org.opensearch.dataprepper.logstash.model.LogstashValueType;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.LinkedList;
+import java.util.ArrayList;
+
+/**
+ * Class to populate Logstash configuration model objects using ANTLR
+ *
+ * @since 1.2
+ */
+
```

Comment:
Please remove extra lines between Javadocs and their classes/methods. I found this one, but there may be others.